tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/upgrade/animated/UpgradeView.java

```diff
@@ -149,39 +149,39 @@ public static void resetPages()
         boolean hasLayout = dataModel.getHasLayout().content();
         boolean hasTheme = dataModel.getHasTheme().content();
         boolean hasExt = dataModel.getHasExt().content();
-        boolean hasWorkspace = dataModel.getConvertLiferayWorkspace().content();
+        /*boolean hasWorkspace = dataModel.getConvertLiferayWorkspace().content();*/
```

Comment:
Does hasHook, hasPortlet, etc return true if the plugins-sdk has been converted to workspace?